apps/server/src/modules/account/services/account.service.ts

```diff
@@ -25,34 +26,51 @@ export class AccountService extends AbstractAccountService {
 	}
 
 	async findById(id: string): Promise<AccountDto> {
+		if (this.configService.get<boolean>('FEATURE_IDENTITY_MANAGEMENT_LOGIN_ENABLED') === true) {
+			return this.accountIdm.findById(id);
+		}
```

Comment:
Can we use an abstract class or interface to have less of these if/else?

The methods called are all the same way, so it is possible. we can switch in the constructor if we use the idm or db